srml/balances/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+
+[package]
+name = "darwinia-balances"
```

Comment:
What's the difference between darwinia-balances and substrate balances?

Reply:
I guess one of the reasons is that the original balance module is not generic enough. It would be better if it support config Moment by parameter_types! macro?

It looks bad for me to introduce a duplicate balance module which cause repeat and maintenance efforts.

Could we submit a PR to substrate balances module to make it generic?
```
	#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug)]
pub struct BalanceLock<Balance, Moment> {		pub struct BalanceLock<Balance, BlockNumber> {
	pub id: LockIdentifier,			pub id: LockIdentifier,
	pub amount: Balance,			pub amount: Balance,
	pub until: Moment,			pub until: BlockNumber,
	pub reasons: WithdrawReasons,			pub reasons: WithdrawReasons,
}		}
```